src/code/fv-components/counter.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import t from '../utilities/translate';
+
+const CountView = ({countTitleText, className, currCount, maxCount}) => {
+
+  const countLabel = t(countTitleText),
```

Comment:
I'm worried clients might not pass in a translatable string here...is there a better way to encourage that?